role-manifest.yml

```diff
@@ -66,6 +66,7 @@ roles:
     release_name: cf-mysql
     provides:
       mysql: {}
+  tags: [indexed]
```

Comment:
We should have a note on _why_ it's indexed. (As opposed to clustered.) I assume it's for migration (having a bootstrap node)?

Reply:
I am willing to add the note. Unfortunately I do not know the _why_ myself.

Reply:
And more info with commit f6c387802a1c761ef0ca848649504f22b2a61d19